app/services/user_service/user_signup.rb

```diff
@@ -0,0 +1,30 @@
+module UserService
+    class UserSignup < ApplicationService
+        def initialize(args)
+            @user_name = args[:user_name]
+            @email = args[:email]
+            @password = args[:password]
+        end
+
+        def call
+            user = User.find_by(email: @email)
+            if user 
```

Comment:
이 부분도 
```suggestion
raise ValidationError, "User already exists" if user
```
이런 식으로 서비스 내에선 error raise만 해주시고
호출하는 부분에서 
```
rescue UserService::UserSignup::ValidationError => e
 실패응답
```
이런 식으로 작성하시는 게 좋을 것 같아요

Reply:
넵 올려주시는 코멘트 계속 이슈에 추가해서 해결하도록 하겠습니다!